### PR TITLE
Interaction

### PR DESCRIPTION
**Pull request by HPovoa**
Refactored the Interaction components for Squirrel.

The main change is that all code will be ran from the Raspberry Pi (including serial communication with the interaction board)

### CODE REVIEW

File: squirrel_interaction/board/src/teleop.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        speakers.
+
+"""
+
+import rospy
+from sensor_msgs.msg import Joy, JointState
+from std_msgs.msg import Float64
+from geometry_msgs.msg import Twist
+from math import degrees, radians
+from raspberry_screen.srv import *
```
Comment (NadiaHG) [inline squirrel_interaction/board/src/teleop.py]: rasberry_screen package does not exists

**Review by ipa-nhg — CHANGES_REQUESTED** [review]
@HPovoa good work! :+1: 
Just a comment about the structure, could we merge all the subpackages into one package? at the moment you have a folder (squirrel_interaccion) and inside some packages with generic names (board, launch, screen and speakers). We have two problems: this is not following the structure of the repository and we can not use generic names like launch for a package.

I think that make more sense merge everything together in one package squirrel_interaction, where we have the launch files and the nodes together.
If it is ok for you I can restructure the packages and you can review my suggestion and test it, ok?

File: squirrel_interaction/board/src/teleop.py (hunk 1 above)
Comment (HPovoa) [reply squirrel_interaction/board/src/teleop.py]: Yes, you can help me restructure the packages. 

I'll get right on the other raspberry_screen problem, must have been a commit issue.

**Comment by ipa-nhg** [comment]
@HPovoa please review https://github.com/HPovoa/squirrel_driver/pull/1 and merge if it is ok for you

**Comment by ipa-nhg** [comment]
@HPovoa please let me know when this is tested and working on the robot to merge the Pull Request
Thanks!

**Comment by HPovoa** [comment]
Hi Nadia,

The code is working on the robot, I tested it yesterday but today I wanted
to do a quick sweep in case i missed something.

There are still changes I want to make, but this is a good time to merge i
think.

I would ask you to review the code where i publish to /joint_states, it is
in board.py, because I am unsure if this is the correct way to do it.

Very best,
Henrique Póvoa

2017-08-10 11:25 GMT+01:00 Nadia Hammoudeh García <notifications@github.com>
:

**Review by ipa-nhg — APPROVED** [review]

**Pull request merged**